Source/santad/Logs/SNTEventLog.m

```diff
@@ -447,7 +447,10 @@ + (instancetype)logger {
         logger = [[SNTProtobufEventLog alloc] init];
         break;
       }
-      default: logger = nil;
+      case SNTEventLogTypeNull: {
+        logger = nil;
```

Comment:
Maybe add a note here explaining why using nil is okay. My understanding is that we are relying on `objc_msgSend` to efficiently ignore the logging messages that will be sent to a nil receiver. This is much cleaner and probably just as efficient as doing a nil check on the return value of `[SNTEventLog logger]` before sending the objc message.

https://github.com/apple-oss-distributions/objc4/blob/488ca1fa9db825a411ba002a7e159bd4aaafc3c4/runtime/Messengers.subproj/objc-msg-x86_64.s#L730